snow/networking/handler/handler.go

```diff
@@ -742,7 +742,7 @@ func (h *handler) handleAsyncMsg(ctx context.Context, msg Message) {
 			h.StopWithError(ctx, fmt.Errorf(
 				"%w while processing async message: %s",
 				err,
-				msg,
+				msg.String(),
```

Comment:
I don't think we actually need to call `.String()` here, `fmt.Errorf` should do it for us. (Same above)